panther_bringup/config/components.yaml

```diff
@@ -0,0 +1,27 @@
+# The uncomment content in this file is the minimal requirement for components configuration file.
```

Comment:
Suggest to add it to panther_description

Reply:
This is good point. I was thinking about this. I put this the bringup because you choose your components at runtime let's say but maybe the default components `components: []` should be in description.

What do you prefer?